tests/Executor/TestClasses/MyContext.php

```diff
@@ -0,0 +1,9 @@
+<?php declare(strict_types=1);
+
+namespace GraphQL\Tests\Executor\TestClasses;
+
+class MyContext
```

Comment:
```suggestion
final class MyContext
```

All classes in `tests`